mirage/scenarios/default.js

```diff
@@ -0,0 +1,9 @@
+export default function(/* server */) {
```

Comment:
yea auto-generated.  

I don't know Mirage all that well, and this is new in 2, so I just left it.  I think it does kind of communicate that we have the version of Mirage that uses this kind of file?  I could go either way honestly.